src/glview/cgal/CGALRenderUtils.cc

```diff
@@ -21,3 +50,23 @@ double calculateLineLineDistance(const Vector3d &l1b, const Vector3d &l1e, const
 	dist_lat=(v2.cross(n)).dot(l2b-l1b)/t;
 	return d;
 }
+
+double calculateSegSegDistance(const Vector3d &l1b, const Vector3d &l1e, const Vector3d &l2b, const Vector3d &l2e, double &dist_lat)
```

Comment:
What is dist_lat?

Reply:
dist_lat is the length of the  lateral portion of  distance from l1 to pt parallel to  the line

Reply:
Sorry, badly formulated question: This argument doesn't seem to be used; is that an oversight?

Reply:
i copied this function from my codes. Many times the caller is also interested in the lateral distance, however not in this case. sounds like you don' t want it for future extensions ;)

Reply:
using this parameter you can e.g. easily determine, if a pt is projecting onto the line.

Reply:
Sounds great for a future extension, but it's, IMO, better to leave out unimplemented arguments, and rather leave a comment in the code describing possible future extensions.